schemas/test/en/test_placeholder_option_label_from_value.json

```
@@ -0,0 +1,266 @@
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "checkbox-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "mandatory-checkbox",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "mandatory-checkbox-answer",
+                                        "mandatory": true,
+                                        "options": [
+                                            {
+                                                "label": "Head",
+                                                "value": "Head"
+                                            },
+                                            {
+                                                "label": "Body",
+                                                "value": "Body",
+                                                "q_code": "1"
+                                            },
+                                            {
+                                                "label": "Right Arm",
+                                                "value": "Right Arm",
+                                                "q_code": "2"
+                                            },
+                                            {
+                                                "label": "Left Arm",
+                                                "value": "Left Arm",
+                                                "q_code": "3"
+                                            }
+                                        ],
+                                        "type": "Checkbox"
+                                    }
+                                ],
+                                "id": "mandatory-checkbox-question",
+                                "title": "When you had your accident, where did you sustain injuries?",
+                                "type": "General"
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "recovery-question-checkbox-block",
+                            "question": {
+                                "id": "recovery-question-checkbox",
+                                "title": {
+                                    "text": "How long did it take to recover from the injury to your {body_part}?",
```

Just for consistency as the other routes has it highlighted.
```suggestion
                                    "text": "How long did it take to recover from the injury to your <em>{body_part}</em>?",
```